FrontEnd/src/components/AboutPage/MeetTheTeam/MeetTheTeam.module.scss

```diff
@@ -0,0 +1,53 @@
+.cardsList_container {
```

Comment:
Shouldn't use the underline. Keep it camel cased to be consistent with rest of codebase. Same for the other underlined ones